nixos/modules/services/mail/postgrey.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+with lib; let
+
+  cfg = config.services.postgrey;
+
+in {
+
+  options = {
```

Comment:
Not that I'm aware. For example, setting `service.postgrey.enable = "yes, please"` won't fail until the value is used in a context where a boolean value is expected (e.g., `mkIf`). All entries in the options chapter will also show "unspecified" as the type.